cobigen-angular-client-generation/index.asciidoc

```diff
@@ -0,0 +1,73 @@
+= Generate Angular Client with devonfw
+== Angular client generation with CobiGen
+====
+With the CobiGen code generator you can easily generate components for your Angular application.
+The following tutorial provides an example for using the CobiGen cli.
+You can find more information on https://github.com/devonfw/cobigen/wiki/howto_angular-client-generation
+## Prerequisites
+* devonfw Ide
+
+## Learning goals.
+Here in this tutorial we will learn the following:
+* Create java project
+* CobiGen to generate the Angular source code out of the java files
+
+====
+
+[step]
+--
+restoreDevonfwIde(["java", "mvn", "vscode"])
+--
+
+For this tutorial an installation of the devonfw IDE is required, which was already set up for you. The next step is to install the CobiGen CLI.
+[step]
+--
+installCobiGen()
+--
+
+CobiGen uses the backend classes for generating your code used in the Angular application. So in the next steps you will create a java project, create a simple java class.
```

Comment:
Please check if you could explain this in more detail.

Reply:
> Please check if you could explain this in more detail.

More explanation is added .